week-3/reading-list/script.js

```diff
@@ -20,4 +20,26 @@ const books = [
     bookCoverImage: "https://blackwells.co.uk/jacket/l/9780135957059.jpg",
   },
 ];
-
+// document.body.style.width = "100vh";
+const unorderList = document.getElementById("reading-list");
+books.forEach((book) => {
+  const newlist = document.createElement("li");
+  newlist.style.width = "90%";
+  const titlePar = document.createElement("p");
+  const authorPar = document.createElement("p");
+  const backImage = document.createElement("img");
+  backImage.style.width = "30%";
+  backImage.style.height = "50%";
+  titlePar.textContent = book.title;
+  authorPar.textContent = book.author;
+  backImage.src = book.bookCoverImage;
+  newlist.appendChild(titlePar);
+  newlist.appendChild(authorPar);
+  newlist.appendChild(backImage);
+  unorderList.append(newlist);
+  if (book.alreadyRead == true) {
+    newlist.style.background = "green";
```

Comment:
Similar to the comment above, it would make your code clearer if you added a class (such as `.read` or `.unread`) to the newly created `li` element. Then the styles for these classes could be created in `style.css`, and all style information would be in one place